src/librustc_codegen_ssa/mir/block.rs

```diff
@@ -784,17 +787,19 @@ impl<'a, 'tcx, Bx: BuilderMethods<'a, 'tcx>> FunctionCx<'a, 'tcx, Bx> {
             }
         }
 
-        // Makes a seg FIXIT
-        //if !bx.tcx().def_path_str(self.instance.def_id()).contains("drop_in_place") {
-        //    use ykpack::BLOCK_LABEL_PREFIX;
-        //    let lbl_name = CString::new(format!(
-        //            "NEW_{}:{}:{}",
-        //            BLOCK_LABEL_PREFIX,
-        //            bx.cx().tcx().symbol_name(self.instance),
-        //            bb.index()
-        //    )).unwrap();
-        //    bx.add_yk_block_label(lbl_name);
-        //}
+        if bx.tcx().sess.opts.cg.tracer.sir_labels() &&
+            !bx.tcx().def_path_str(self.instance.def_id()).contains("drop_in_place") &&
```

Comment:
I think these should be exact equality checks so as to not skip user functions that may contain "drop_in_place".

IIRC, the two offenders are `core::drop_in_place` and `std::drop_in_place`, but might be wrong.